dipy/reconst/dki.py

```diff
@@ -2579,16 +2731,13 @@ def split_dki_param(dki_params):
 
 
 common_fit_methods = {'WLS': ls_fit_dki,
+                      'LS': ls_fit_dki,
                       'OLS': ls_fit_dki,
-                      'NLS': nlls_fit_tensor,
```

Comment:
Okay, if you would like all of these recently added acronyms back in, I can add them back (although I would suggest not to add RT for RESTORE back in, because that is quite silly).

Reply:
>  I would suggest not to add RT for RESTORE back in, because that is quite silly

this is one opinion among many 😄 

In any case, if we decided to remove it, it needs a deprecation cycle to avoid breaking many code. I think it costs nothing to let it for now.

Reply:
wasn't RT was only added recently, at the same time that a RESTORE test would have gone over 79 characters without the acronym "RT" (which I found to be suspicious)

Easier for me to just add "RT" back in, but it is so vague - RESTORE was broken in DiPy until relatively recently anyway, because you were all defining the weights wrong in the Jacobian #2730  ;)

Reply:
I am fine with removing or not 'RT'.

Reply:
why is UWLLS the acronym for weighted least squares? Surely UW would be understood as "un-weighted" (see the docstring for `DiffusionKurtosisModel` and elsewhere, showing that this corresponds to _weighted_ least squares in DiPy)

Is this an error? If it is, this is exactly the sort of thing that inspired me to remove the superfluous recently added acronyms...

Reply:
@RafaelNH any thoughts? Thanks

Reply:
yes that was for un-weighted - being connected to WLS is indeed a typo. That option was included because there was a paper using that acronym for OLS. But I agree that we can remove that option given that it can create some confusion